jvm/src/main/scala/uk/gov/homeoffice/drt/db/serialisers/FlightSerialiser.scala

```diff
@@ -14,7 +14,24 @@ import java.sql.Timestamp
 
 
 trait FlightJsonFormats extends DefaultJsonProtocol {
-  implicit val flightJsonFormat: RootJsonFormat[Predictions] = jsonFormat2(Predictions)
+  implicit object PredictionsJsonFormat extends RootJsonFormat[Predictions] {
+    override def read(json: JsValue): Predictions = json match {
+      case JsObject(fields) =>
+        val lastUpdated = fields.getOrElse("lastUpdated", fields.getOrElse("lastChecked", JsNumber(0))).convertTo[Long]
+        val predictionValues = fields.get("predictions") match {
+          case Some(JsObject(predFields)) =>
```

Comment:
match is not covering None . So I guess you happy it throw error for that scenario ?

Reply:
Yeah it's a required field so should always be there